www/widgets/rt.js

```diff
@@ -99,7 +113,33 @@ static tkinter(options) {
         }
 
         bind(key, fn) {
-            // this.tk.addEventListener(...)
+            if (['<Up>', '<Down>', '<Left>', '<Right>'].indexOf(key) !== -1) {
```

Comment:
Is typing in the URL bar supposed to work? I tried building your branch just now and it didn't seem to.

Reply:
It doesn't yet. I would like to add support, but I need to make a UI to explain that the web version is limited to same-origin.

Reply:
URL is now configurable, but has to stay on the "browser.engineering" domain. Avoids having to explain this stuff.